scripts/display-travis-status.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const debug = require('debug')('display_travis_status')
+const pollTravis = require('../lib/pollTravis')
+const enabledRepos = ['citgm', 'readable-stream', 'nodejs.org']
+
+module.exports = function (app) {
+  app.on('pull_request.opened', (event) => {
+    const owner = event.repository.owner.login
+    const repo = event.repository.name
+    if (!~enabledRepos.indexOf(repo)) return
```

Comment:
this WAS `enabledRepos.includes(repo)`.. but then I realized I'm getting too comfortable with the nighty version and it isn't available in v4/5.. so I switched it to the old hackery way.